server/src/infra/repositories/media.repository.ts

```diff
@@ -23,10 +24,25 @@ export class MediaRepository implements IMediaRepository {
   }
 
   async resize(input: string | Buffer, output: string, options: ResizeOptions): Promise<void> {
-    await sharp(input, { failOn: 'none' })
+    let colorspace = options.wideGamut ? 'p3' : 'srgb';
```

Comment:
I'm not sure I like the variable 'wideGamut'. There are more profiles than just P3.

Reply:
What other web-compatible wide-gamut profiles are there?

Reply:
`preserveColorspace` is misleading because it doesn't actually keep the image's colorspace.

I originally had a drop-down with sRGB and P3, but a slider makes for better UX in this case since these will continue to be the only real options for the foreseeable future. I can change the internal variable to store the color profile instead of being a boolean to handle the case where another viable option shows up.

Reply:
That is a fine solution to me.